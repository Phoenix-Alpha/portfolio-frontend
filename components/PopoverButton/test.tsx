import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { PopoverButton } from '.'

describe('<PopoverButton />', () => {
  it('should renders correct <PopoverButton />', () => {
    const wrapper = shallow(<PopoverButton />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
