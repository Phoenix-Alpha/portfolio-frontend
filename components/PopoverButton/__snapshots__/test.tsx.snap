// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopoverButton /> should renders correct <PopoverButton /> 1`] = `
<button
  className="inline-flex relative items-center ml-1"
  onClick={[Function]}
  onMouseEnter={[Function]}
/>
`;
